C-Plus-Plus/cp/Three_Sum.cpp

```diff
@@ -0,0 +1,70 @@
+/*
+Description :
+    
+	Three Sum Problem - In this problem, user will provide an array and we
+	                    have find wether there is any "three" numbers are 
+						present in the array who's sum is equal to the target 
+						provided.If it is present then it will show true else 
+						it will show false.
+
+*/
+
+
+#include<bits/stdc++.h>
+using namespace std;
+
+int main()
+{
+	int n; // n= size of an array
```

Comment:
Give a good naming convention

Reply:
What I assume that naming convention depends upon user and it's like maybe some variable name you find inappropriate whereas other find good . That's why I have added the comment on that n = size of array

Reply:
**What you assume don't work here.** For **better User-interface**, it's** necessary** to do the changes as suggested.